corehq/apps/es/tests/test_case_search_es.py

```diff
@@ -543,3 +545,80 @@ def test_starts_with_query(self):
             "starts-with(ssn, '100')",
             ['c5', 'c6', 'c2']
         )
+
+    def test_user_input_forward_timezone_adjustment(self):
+        """
```

Comment:
love these tests and descriptions!

Reply:
the only nit I have is that each assertion / scenario should be its own test. that way if one of the scenarios fail at the beginning, it doesn't cause the remaining scenarios to fail by default